src/main/scala/org/apache/spark/streamdm/classifiers/trees/HoeffdingTree.scala

```diff
@@ -113,10 +117,11 @@ class HoeffdingTree extends Classifier {
     val outputSpec = espec.outputFeatureSpecification(0)
     val numClasses = outputSpec.range()
     model = new HoeffdingTreeModel(espec, numericObserverTypeOption.getValue, splitCriterionOption.getValue(),
-      growthAllowedOption.isSet(), binaryOnlyOption.isSet(), numGraceOption.getValue(),
+      true, binaryOnlyOption.isSet(), numGraceOption.getValue(),
```

Comment:
Why growthAllowedOption is hard-coded as true?

Reply:
This is unused parameter, and should be removed. I made in hard-coded during the test. This will be fixed soon for the next commit.

By default, this parameter is turned FALSE, so that it makes the tree unable to grow.

Reply:
Thanks for the clarification